recipes/wilzegers_autotest/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "0.1":
+    url: "https://gitlab.com/wilzegers/autotest/-/archive/master/autotest-master.tar.gz"
+    sha256: "b0801e5be7600edc5d527c7a0e88094f1d2db875c7c8b533dd2c7b90bd9f0063"
```

When a upstream project lacks a release, use specific commits to create reproducible packages.
```suggestion
  "cci.20200921":
    url: "https://gitlab.com/wilzegers/autotest/-/archive/501c6e96e41c3fb60ac667c17a405b56b2527963/autotest-501c6e96e41c3fb60ac667c17a405b56b2527963.tar.gz"
    sha256: "aa4f8fca7cb113e5452c0c09a2dfc1e73989742fccb490dab1aa8c127ca10979"
```

---

Nice catch. I created https://gitlab.com/wilzegers/autotest/-/issues/1